packages/FormElement/stories/examples/AccessibilityExample.js

```diff
@@ -37,7 +37,18 @@ const ExampleStory = () => {
             text for extra panel for questionnaires. Example text for extra panel for questionnaires
           </span>
         </FormElement.Instructions>
-        <Input onChange={handleChange} value={value} placeholder="Form placeholder" />
+        <FormElement.Content>
+          {({ idForLabel, ariaDescribedBy }) => (
+            <Input
+              id={idForLabel}
+              onChange={handleChange}
+              value={value}
+              placeholder="Form placeholder"
+              aria-describedby={ariaDescribedBy}
+              hasError={Boolean(errorText.length)}
```

Comment:
Proposing we leave the `hasError` prop on the children to be set by the consumer rather than have the FormElement do it. That way if they have a number of children (more than one input for example) and they want to highlight only one they can.